ApiCalls.js

```diff
@@ -28,78 +32,106 @@ export const addFlyToAPI = async (flyData) => {
 
 export const addUpdatedFly = async (flyData) => {
   try {
-    const response = await fetch(`https://my-fly-box-api.herokuapp.com/api/v1/flies/${flyData.id}`, {
-      method: 'PATCH',
-      headers: {
-        'Content-Type': 'application/json;charset=utf-8'
-      },
-      body: JSON.stringify(flyData.attributes),
-    });
+    const response = await fetch(
+      `https://my-fly-box-api.herokuapp.com/api/v1/flies/${flyData.id}`,
+      {
+        method: "PATCH",
+        headers: {
+          "Content-Type": "application/json;charset=utf-8",
+        },
+        body: JSON.stringify(flyData.attributes),
+      }
+    );
     const flyDataResponse = await response.json();
     return flyDataResponse;
   } catch (error) {
     console.error(error);
   }
-}
+};
 
 export const deleteFly = async (flyId) => {
-    try {
-      const response = await fetch(
-        `https://my-fly-box-api.herokuapp.com/api/v1/flies/${flyId}`,
-        {
-          method: "DELETE",
-          headers: {
-            "Content-Type": "application/json;charset=utf-8",
-          },
-        }
-      );
-      const deletionResponse = await response.json();
-      console.log(deletionResponse);
-    } catch (error) {
-      console.error(error);
-    }
-  };
+  try {
+    const response = await fetch(
+      `https://my-fly-box-api.herokuapp.com/api/v1/flies/${flyId}`,
+      {
+        method: "DELETE",
+        headers: {
+          "Content-Type": "application/json;charset=utf-8",
+        },
+      }
+    );
+    const deletionResponse = await response.json();
+    console.log(deletionResponse);
+  } catch (error) {
```

Comment:
Thanks for adding this in. This will be useful when we start testing the UX of the application.